cli/sdo/key.go

```diff
@@ -287,3 +297,25 @@ func checkEmptyKeyFields(key KeyFile) error {
 	}
 	return nil
 }
+
+func checkKeyName(s string) error {
+	for _, r := range s {
+        if (r < 'a' || r > 'z') && (r < '0' || r > '9') && r != '-' {
+            return errors.New("Key name can only contain lowercase characters, numbers, and hyphens")
+        }
+    }
+	return nil
+}
+
+func checkKeyCountry(s string) error {
+	err := "Country name must be a 2 Letter Country Code"
+	if len(s) > 2 {
```

Comment:
My understanding is that the country code must be exactly 2 chars, so just check `len(s) == 2`

Reply:
It should be `len(s) != 2`. I'm not sure when or why I changed it to `>` but that is definitely incorrect.